09-intermediate-rails/oauth.md

```diff
@@ -26,74 +29,65 @@ gem "omniauth"
 gem "omniauth-github"
 ```
 
-Save your Gemfile, then head over to your terminal, where you'll need to `$ bundle`. Notice that there's a specific gem for autheticating with Github. Each _provider_ has a small Ruby gem that's responsible for the specifics of how to authenticate with that service.
+Save your Gemfile, then head over to your terminal, where you'll need to `$ bundle`. Notice that there's a specific gem for authenticating with GitHub. Each _provider_ has a small Ruby gem that's responsible for the specifics of how to authenticate with that service.
 
-### Github Credentials
-Each provider requires you to provide some credentials for your application, so they can keep track of which website is authorizing which user. [Login to Github and register a new "application"](https://github.com/settings/applications/new).
+### GitHub Credentials
+Each provider requires you to provide some credentials for your application, so they can keep track of which website is authorizing which user. [Login to GitHub and register a new "application"](https://github.com/settings/applications/new).
 
-![Github Application Registration](./images/github-application.png)
+![GitHub Application Registration](./images/github-application.png)
 
 After registration you will be given a `client id` and a `client secret`:
 
-![Github Application Credentials](./images/github-credentials.png)
+![GitHub Application Credentials](./images/github-credentials.png)
 
-**Note:** These credentials are the equivalent of passwords to your Github account. Keep them safe; never, ever post them in public places or commit them in git.
+**Note:** These credentials are the equivalent of passwords to your GitHub account. Keep them safe; never, ever post them in public places or commit them in git.
 
 #### Storing Credentials
-Because they cannot be committed to git, the Github application credentials must be loaded external of the Rails application. The most common way to do this is to create a file called `.env` in the project root. Dot files (called that because they start with a dot) are typically excluded from version control and are well suited for secrets. __NEVER COMMIT A `.env` TO GIT!__
+Because they cannot be committed to git, the GitHub application credentials must be loaded external of the Rails application. The most common way to do this is to create a file called `.env` in the project root. Dot files (called that because they start with a dot) are typically excluded from version control and are well suited for secrets. __NEVER COMMIT A `.env` TO GIT!__  (This also means if you are working with a partner, you'll need to share this information since it won't be be in git)
```

Comment:
Should we talk a little bit about environment variables, as they'll need that when they deploy to Heroku. 

That also helps explain why `.dotenv-rails` is in the `:development` group.